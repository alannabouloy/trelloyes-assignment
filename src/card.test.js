import React from 'react';
import ReactDOM from 'react-dom';
import Card from './card';
import renderer from 'react-test-renderer';

//Smoke Test
it ('renders without crashing', () => {
    //create a DOM element to render the component
    const div = document.createElement('div');

    //render the component. 

    ReactDOM.render(<Card/>, div );

    //clean up code
    ReactDOM.unmountComponentAtNode(div);
});

//Snapshot Test
it('renders the UI as expected', () => {
    const tree = renderer
        .create(<Card title='test' content='testing'/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});