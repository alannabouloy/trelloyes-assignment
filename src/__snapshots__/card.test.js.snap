// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="Card"
>
  <h3>
    test
  </h3>
  <p>
    testing
  </p>
</div>
`;
